server/views/partials/expectedArrivalCardMacro.njk

```diff
@@ -53,7 +53,7 @@
                 {% set hasPhoto = arrival.matchType in ['SINGLE_MATCH', 'COURT_RETURN'] %}
                 <img 
                   src="{{ '/prisoners/' + arrival.potentialMatches[0].prisonNumber + '/image' if hasPhoto else '/assets/images/placeholder-image.png' }}" class="app-card__image" 
```

Comment:
The `alt` is currently set on the `arrival.matchType` The assumption is that all other match types currently don't have a photo and therefore it is set to 'No photo available`

Reply:
I'm not sure about the logic here!
Is it agreed by UCD that all other match types won't have a photo?